### DISCUSSION TITLE
Release 0.0.5

### DISCUSSION THREAD

**Discussion by Ly-sec** [Ly-sec/PalBot#17] [General]
Fully rewritten the bot,
Added a better structure to it,
Multiple rcon connections are  now possible (should lead to less errors),
Status fixed (once again),
Korean names should be visible now

<hr /><em>This discussion was created from the release <a href='https://github.com/Ly-sec/PalBot/releases/tag/release-0.0.5'>Release 0.0.5</a>.</em>

**Comment by klos1908** [comment]
Executing task: npm install 

npm : The term 'npm' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ npm install
+ ~~~
    + CategoryInfo          : ObjectNotFound: (npm:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 

 *  The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command npm install" terminated with exit code: 1. 
 *  Terminal will be reused by tasks, press any key to close it. 


what i must do???

**Reply by AlexisD09** [reply]
Hey ! You have to install nodejs

**Comment by ThrowU** [comment]
Why can I connect to my server but I don't have a whitelist?
Because I'm the server host?

**Comment by Ly-sec** [comment]
> Why can I connect to my server but I don't have a whitelist? Because I'm the server host?

You would have to add your steamId to the whitelist.

**Reply by ThrowU** [reply]
I didn't add but I got in and didn't get kicked.

**Reply by Ly-sec** [reply]
> I didn't add but I got in and didn't get kicked.

Did you activate the whitelist?

**Reply by ThrowU** [reply]
https://img2.pic.in.th/pic/Screenshot-2024-02-09-232054.png

i put 12345 in Whitelist but i also can connect

**Comment by CommanderG1Sniper** [comment]
Hi

Bot is working as intended however its not actually kicking the player even when i get this message

RCON Client connected
RCON Client disconnected
RCON Client connected
Not all steamIds from steamIds match with whitelistArray.
Non-matching steamIds: [ '76561198867******' ]
Kicking player with non-matching steamId: 76561198867******
RCON Client disconnected

any help is appreciated

**Reply by CommanderG1Sniper** [reply]
@Ly-sec
